local-cli/util/Config.js

```diff
@@ -25,26 +25,35 @@ let cachedConfig = null;
  * error will be thrown.
  */
 const Config = {
-  get(cwd, defaultConfig) {
-    if (cachedConfig) {
```

Comment:
so we are no longer doing caching? What's the possible side effects of this change?